tools/sapp/sapp/interactive.py

```diff
@@ -1403,12 +1407,13 @@ def _create_issue_output_string(
         return "\n".join(
             [
                 f"Issue {issue.id}",
-                f"    Code: {issue.code}",
-                f" Message: {issue.message}",
-                f"Callable: {issue.callable}",
-                f" Sources: {sources_output if sources_output else 'No sources'}",
-                f"   Sinks: {sinks_output if sinks_output else 'No sinks'}",
-                (f"Location: {issue.filename}" f":{issue.location}"),
+                f"            Code: {issue.code}",
+                f"         Message: {issue.message}",
+                f"        Callable: {issue.callable}",
+                f"         Sources: {sources_output if sources_output else 'No sources'}",
+                f"           Sinks: {sinks_output if sinks_output else 'No sinks'}",
+                f"        Location: {issue.filename}" f":{issue.location}",
+                f"Min Trace Length: Source ({issue.min_trace_length_to_sources}) | Sink ({issue.min_trace_length_to_sinks})",
```

Comment:
Ugh, I'm really sorry, the linter on our internal tests is a bit more strict. It doesn't like that these lines (1413 and 1416) are > 80 characters. Can you wrap/refactor in some way?

Reply:
no worries, I'll reformat it to be more compliant 😄